python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py

```diff
@@ -243,12 +257,32 @@ class Meta:
             GrapheneTerminateRunSuccess,
             GrapheneTerminateRunFailure,
             GrapheneRunNotFoundError,
-            GrapheneUnauthorizedError,
+            GrapheneTerminateRunUnauthorizedError,
```

Comment:
if we add this new type, we should be sure that the frontend knows how to handle it